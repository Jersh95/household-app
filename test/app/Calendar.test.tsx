import Calendar from "../../app/calendar/Calendar";
import renderer from 'react-test-renderer';

describe('Calendar', () => {
    it('should render', () => {
        const tree = renderer.create(<Calendar/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});