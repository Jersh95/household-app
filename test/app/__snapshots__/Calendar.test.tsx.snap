// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calendar should render 1`] = `
<View>
  <Text>
    Calendar
  </Text>
</View>
`;
